### DISCUSSION TITLE
Sendig the Firmware

### DISCUSSION THREAD

**Discussion by Maxklui** [bangcorrupt/hacktribe#190] [Q&A] [answered]
Hello

I keep getting messages on facebook including how to get to the hacktribe firmware. I used the old hack method with Python at the time, but that no longer works. I just recently tried it and sent someone and they tried to install the firmware but it said "Invalid File". I used the old method with Python that I used when the hack came out. I wanted to ask if someone would allow me to update the firmware for both devices (blue and red). would be super helpful. also want to try to create something that will allow people to download and install the firmware very easily.

Thanks in advance

Best greetings

**Comment by bangcorrupt** [comment, marked as answer]
"Invalid File" usually means the file header is wrong for the device.

**Reply by Maxklui** [reply]
@bangcorrupt 

Is there an possibilty to fix this? because this method worked for me the first time.
